analysis/benchmarking-functions/R/aws_copy_samples.R

```diff
@@ -0,0 +1,38 @@
+aws_copy_samples <- function(local_dir, s3_dir, samples, tools) {
+  
+  ## need to add in check for tools 
+  
+  includes <- stringr::str_glue("--include \"{samples}*\"")
+  
+  for (tool in tools) {
+    local_tool_dir = file.path(local_dir, tool)
+    s3_tool_dir = paste(s3_dir, paste(tool, 'quant/', sep = '-'), sep = '/')
+    dir.create(local_tool_dir, recursive = TRUE, showWarnings = FALSE)
+    
+    if (tool == 'alevin') {
+      sync_call <- paste('aws s3 cp', s3_tool_dir, local_tool_dir, 
```

Comment:
So there is a reason I chose to switch it over to using `cp` instead and that's because last time when I was using `sync`, it wasn't recognizing that when I had an old version of a sample that was not run fully it didn't copy over it (that was the issue with that weird sample last time), even though the files were different. So to make sure that we are always working with the correct files from S3 from that sample and don't have that mix up, switching to `cp` made the most sense to me.

Reply:
Ah yes, that was annoying. Seems a reasonable change.